### PR TITLE
FIREFLY-1349: Add firefly icon in JupyterLab launcher

### PR DESCRIPTION
**Pull request by jaladh-singhal**
Fixes [FIREFLY-1349](https://jira.ipac.caltech.edu/browse/FIREFLY-1349)

## Testing

Checkout this branch locally and make sure that you have [development installation of firefly extension](https://github.com/Caltech-IPAC/jupyter_firefly_extensions?tab=readme-ov-file#development-install) in your python environment. (If you previously installed jupyter lab in development mode then you just need to run last command `jlpm run build`). Now open jupyter lab and In the launcher, firefly icon should show instead of blank space.

### CODE REVIEW

**Comment by gpdf** [comment]
Just a suggestion: could you split this into two commits?  One for the actual SVG file, with a commit message that includes something like "temporary bitmap SVG", and then one for the code?  It will help preserve a reminder that we don't intend this to be the last word.

File: style/fftools-logo.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+    <!-- base64-encoded 128x128 png image of firefly logo -->
```
Comment (jaladh-singhal) [inline style/fftools-logo.svg]: @gpdf yes for that reminder, I've put a comment in svg file itself. I can change the text for more clarity?

Comment (jaladh-singhal) [reply style/fftools-logo.svg]: Github shows "rich diff" by default, you'll need to toggle to "source diff" by using toggle buttons on top right of this file.

**Review by stargaser — APPROVED** [review]
I built the extension from the branch and tested it. The icons appear as is desired. Looks good to me!

**Review by robyww — APPROVED** [review]
Looks good.

**Pull request merged**